docs/OVERVIEW.md

```diff
@@ -0,0 +1,36 @@
+
+# Ship-it! Overview
+
+This document provides a summary of Velocity's Kubernetes deployment tool Ship-it. Documentation pretaining to registering and deploying microservices with Ship-it is [here](./REGISTER.md).
+
+## Background
```

Comment:
We should say something about the GitOps model here with a link through to the article about it.